.github/workflows/maven-publish.yml

```diff
@@ -15,18 +15,16 @@ jobs:
       packages: write
 
     steps:
-    - uses: actions/checkout@v3
```

Comment:
@demurelebowski please add changes related to maven scripts configuration as a separate PR